.github/workflows/R-CMD-check.yaml

```diff
@@ -2,9 +2,9 @@
 # Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   push:
-    branches: [main, master]
+    branches: [Trace]
```

Comment:
revert this, or remove [main, master], then it will run on all jobs

Reply:
get it